fleet-community-playbook.yml

```diff
@@ -0,0 +1,32 @@
+site:
+  title: Fleet
+  url: /
+  start_page: next@fleet::index.adoc
+
+content:
+  sources:
+    - url: ./
+      start_paths: [versions/v0.13, versions/v0.12]
+      branches: HEAD
+
+# Description: SUSE UI bundle
+ui:
+  bundle:
+    url: https://github.com/SUSEdoc/dsc-style-bundle/blob/main/default-ui/ui-bundle.zip?raw=true
+    snapshot: true
+  supplemental_files: ./dsc-style-bundle/supplemental-files/rancher
```

Comment:
This is the community playbook, but the product supplemental files are being referenced.